modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ListTypeDefinition/ListTypeTable.tsx

```diff
@@ -0,0 +1,218 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+// @ts-ignore
+
+import {
+	FrontendDataSet,
+	IFrontendDataSetProps,
+} from '@liferay/frontend-data-set-web';
+import React, {useEffect, useState} from 'react';
+
+import {IModalState} from './ListTypeEntriesModal';
+
+interface IProps {
+	pickListId: number;
+}
+
+interface Props {
```

Comment:
@cgmonte  I think you can avoid creating the interfaces `Props` and `Name` and just declare the format you need directly on line 44.

Reply:
I redeclared this in a more concise way. Se [here](https://github.com/liferay-objects/liferay-portal/blob/c6092778f5a0d40467857d42b38078662558bcbe/modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ListTypeDefinition/ListTypeTable.tsx#L29).